src/cli/commands/force/lightning/lwc/test/ui/mobile/run.ts

```diff
@@ -107,7 +101,6 @@ export class Run extends SfdxCommand {
             );
         }
 
-        CommonUtils.startCliAction(messages.getMessage('runningUtamTest'));
```

Comment:
UTAM Run command is currently the only command where its output is printed out to VSCode output window via SFDX VSCode Extension. Moreover, `startCliAction` causes an animated text-based spinner to be printed on `stdout`.

The problem is that, on some platforms like Windows, as `startCliAction` spins the text-based spinner, it causes new lines to be sent to `stdout` repeatedly. This spinner doesn't really add any value so the easiest fix is to just remove it from here.